rotkehlchen/chain/evm/decoding/aura_finance/balances.py

```diff
@@ -0,0 +1,107 @@
+import logging
+from collections import defaultdict
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.structures.balance import Balance, BalanceSheet
+from rotkehlchen.chain.ethereum.interfaces.balances import BalancesSheetType, ProtocolWithBalance
+from rotkehlchen.chain.ethereum.utils import token_normalized_value_decimals
+from rotkehlchen.chain.evm.constants import DEFAULT_TOKEN_DECIMALS
+from rotkehlchen.chain.evm.contracts import EvmContract
+from rotkehlchen.chain.evm.decoding.aura_finance.constants import CPT_AURA_FINANCE
+from rotkehlchen.chain.evm.decoding.aura_finance.utils import get_aura_pool_price
+from rotkehlchen.chain.evm.tokens import get_chunk_size_call_order
+from rotkehlchen.errors.misc import RemoteError
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.inquirer import Inquirer
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+if TYPE_CHECKING:
+    from rotkehlchen.assets.asset import EvmToken
+    from rotkehlchen.chain.evm.decoding.decoder import EVMTransactionDecoder
+    from rotkehlchen.chain.evm.node_inquirer import EvmNodeInquirer
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class AuraFinanceBalances(ProtocolWithBalance):
+    def __init__(
+            self,
+            evm_inquirer: 'EvmNodeInquirer',
+            tx_decoder: 'EVMTransactionDecoder',
+    ):
+        super().__init__(
+            tx_decoder=tx_decoder,
+            counterparty=CPT_AURA_FINANCE,
+            evm_inquirer=evm_inquirer,
+            deposit_event_types={
+                (HistoryEventType.DEPOSIT, HistoryEventSubType.DEPOSIT_ASSET),
+            },
+        )
+
+    def query_balances(self) -> BalancesSheetType:
+        balances: BalancesSheetType = defaultdict(BalanceSheet)
+        addresses_with_activities = self.addresses_with_activity(event_types={
```

Comment:
You should decide. Do you use `addresses_with_activity` or `addresses_with_deposits`. If you don't use the latter then you should not initialize the `deposit_event_types` in the constructor. As these type/subtype combinations are what defines what `addresses_with_deposits` returns.

Waht's worse you name the variable `addresses_with_deposits` which makes it confusing.